react-popper.d.ts

```diff
@@ -1,6 +1,7 @@
+import * as React from "react";
+import PopperJS from "popper.js";
```

Comment:
I would actually use `import * as PopperJs from 'popper.js'` as with the new version of Popper the TS compiler seems to complain, see https://github.com/souporserious/react-popper/issues/101.